components/msg/compositor_msg.rs

```diff
@@ -36,19 +36,24 @@ impl FrameTreeId {
 }
 
 #[derive(Clone, PartialEq, Eq, Copy, Hash, Deserialize, Serialize, HeapSizeOf)]
-pub struct LayerId(pub usize, pub u32);
+pub struct LayerId(pub usize, pub u32, pub u32);
```

Comment:
Definitely. I'll see if I can fix https://github.com/servo/servo/issues/2010 too, which should remove the need for the third element in the struct.